index.html

```diff
@@ -30,56 +30,84 @@
+            </g></svg>
+          </span>
+          <h4 class="main-small-heading">Cats</h4>
+        </div>
+      <div class="dogs-cats-birds-rabbits-box">
+          <span class="u-icon u-icon-circle u-icon-3">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" style="height:78px; width:78px;"id="svg-8e0b" class="u-svg-content"><g><path d="m123.156 16.674a1.75 1.75 0 0 0 -1.407-2.3c-.395-.056-9.285-1.3-16.484-.268a21.02 21.02 0 0 0 -2.649-3.037c-5.463-5.152-13.506-7.256-22.651-5.927-9.4 1.368-24.714 7.221-21.62 31.049 2.772 21.348-20.726 47.027-52.382 57.243a1.75 1.75 0 0 0 -.143 3.278c3.581 1.51 10.87 1.754 19.61.667 6.934 7.267 18.049 11.189 31.49 11.3v9.494c-2.493.631-4.772 1.315-5.855 1.65a1.75 1.75 0 0 0 1.036 3.344c4.35-1.347 10.318-2.927 12.779-2.848a1.75 1.75 0 1 0 .074-3.5h-.179a24.158 24.158 0 0 0 -4.355.553v-8.743q3.78-.148 7.762-.678c1.184-.16 2.342-.343 3.488-.537v3.095c-2.493.631-4.774 1.314-5.855 1.649a1.75 1.75 0 1 0 1.036 3.343c4.35-1.347 10.319-2.929 12.779-2.847a1.75 1.75 0 1 0 .074-3.5 22.725 22.725 0 0 0 -4.534.552v-2.947c22.517-4.677 37.074-16.51 44.386-36.026 5.377-14.35-.1-32.665-8.426-38.8 3.397-2.568 9.57-8.028 12.026-15.259zm-6.878 52.836c-7.653 20.427-23.085 31.541-48.563 34.976-16.052 2.164-29.61-.653-37.881-7.749 13.428-2.246 29.605-7.526 35.509-15.987a1.75 1.75 0 0 0 -2.871-2c-5.405 7.75-22.272 13.136-36.595 15.043a74.388 74.388 0 0 1 -12.564.723c30.401-12.016 51.341-36.916 48.502-58.776-2.572-19.809 8.724-25.69 18.654-27.135 8.065-1.175 15.077.607 19.745 5.012 4.479 4.228 6.584 10.735 5.926 18.323a1.748 1.748 0 0 0 1.1 1.778c7.36 2.923 14.633 20.861 9.038 35.792zm-6.535-40.939a26.869 26.869 0 0 0 -2.525-11.171 64.97 64.97 0 0 1 11.8.185c-2.318 4.88-6.497 8.77-9.275 10.986z"></path><path d="m109.066 71.271a46.244 46.244 0 0 1 -3.458 6.314 1.75 1.75 0 1 0 2.926 1.921 49.765 49.765 0 0 0 3.718-6.784 1.75 1.75 0 0 0 -3.186-1.451z"></path><path d="m87.9 16.349a4.75 4.75 0 1 0 4.75 4.75 4.756 4.756 0 0 0 -4.75-4.75zm0 6a1.25 1.25 0 1 1 1.25-1.25 1.251 1.251 0 0 1 -1.25 1.25z"></path>
+           </g></svg>
+          </span>
+          <h4 class="main-small-heading">Birds</h4>
+      </div>
+      <div class="dogs-cats-birds-rabbits-box">
+          <span class="u-icon u-icon-circle u-icon-4">
```

Comment:
Te klasy w spanie są niepotrzebne. Jak kopiujesz coś to staraj się zawsze sprzątać syf. W tamtym projekcie te klasy pewnie miały sens i coś robiły. Tutaj nie robią nic.

Reply:
Poprawione